packages/web-app-files/src/views/FilesDrop.vue

```diff
@@ -74,29 +34,57 @@
 </template>
 
 <script>
-import vue2DropZone from 'vue2-dropzone'
-import { mapGetters } from 'vuex'
-import Mixins from '../mixins.js'
+import { mapActions, mapGetters } from 'vuex'
 import { DavProperties, DavProperty } from 'web-pkg/src/constants'
 import { linkRoleUploaderFolder } from '../helpers/share'
 import { createLocationOperations, createLocationPublic } from '../router'
 
+import FileUpload from '../components/AppBar/Upload/FileUpload.vue'
+import {
+  getCurrentInstance,
+  onMounted,
+  onUnmounted
+} from '@vue/composition-api/dist/vue-composition-api'
+import { useUpload } from 'web-runtime/src/composables/upload'
+
 export default {
   components: {
-    vueDropzone: vue2DropZone
+    FileUpload
+  },
+  setup() {
+    const instance = getCurrentInstance().proxy
+    const uppyService = instance.$uppyService
+
+    onMounted(() => {
+      uppyService.$on('filesSelected', instance.onFilesSelected)
```

Comment:
Currently they won't crash because both components don't co-exist at the same time. But it might be a scenario we have to watch out for in the future.